### PR TITLE
Startup boilerplate: fix line breaks

### PR DESCRIPTION
**Pull request by HybridDog**

### CODE REVIEW

**Comment by everamzah** [comment]
That's nice, it's been wrapping for a long time.  The common solution of "resizing X terminal emulator" doesn't apply as much when you're running from a Virtual Terminal and have limited font/resolution choices.

I notice now that this appears only once you've started a world.  It'd be nice to see it as soon (and before any other message) as minetest is launched.

**Comment by HybridDog** [comment]
> It'd be nice to see it as soon (and before any other message) as minetest is launched.

l agree, but it's not in the scope of this PR, l think.

File: src/server.cpp [hunk 1]
```diff
@@ -443,7 +443,8 @@ void Server::start(Address bind_addr)
 	m_thread->start();
 
 	// ASCII art for the win!
-	actionstream
+	// http://patorjk.com/software/taag/#p=display&h=1&f=Small%20Shadow&t=minetest
```
Comment (Thomas--S) [inline src/server.cpp]: IMHO, this is the correct link:
http://patorjk.com/software/taag/#p=display&h=1&f=Graffiti&t=minetest

Comment (HybridDog) [inline src/server.cpp]: thanks, l should have tested the link before taking it from the old PR

**Comment by sofar** [comment]
Can we have a reference to `figlet` and the font used instead of a hyperlink to an ad-supported website instead?

We could at least link to figlet.org instead.

**Comment by paramat** [comment]
#3817

File: src/server.cpp [hunk 2]
```diff
@@ -443,7 +443,8 @@ void Server::start(Address bind_addr)
 	m_thread->start();
 
 	// ASCII art for the win!
-	actionstream
+	// http://patorjk.com/software/taag/#p=display&h=1&f=Graffiti&t=minetest
+	std::cout
```
Comment (rubenwardy) [inline src/server.cpp]: This should probably be cerr

**Comment by HybridDog** [comment]
l don't longer experience this problem since l changed my default terminal window size.

**Comment by paramat** [comment]
Yeah i feel it is best to just resize terminal.

**Pull request closed without merging**